pkg/internal/export/otel/metrics.go

```diff
@@ -51,17 +51,17 @@ type MetricsConfig struct {
 	MetricsProtocol Protocol `yaml:"-" env:"OTEL_EXPORTER_OTLP_METRICS_PROTOCOL"`
 
 	// InsecureSkipVerify is not standard, so we don't follow the same naming convention
-	InsecureSkipVerify bool `yaml:"insecure_skip_verify" env:"OTEL_INSECURE_SKIP_VERIFY"`
+	InsecureSkipVerify bool `yaml:"insecure_skip_verify" env:"BEYLA_OTEL_INSECURE_SKIP_VERIFY"`
```

Comment:
Some `OTEL_...` variables append `BEYLA_` because they are not really standards. For example this one or the sampling configuration.